Terminal.Gui/Views/DatePicker.cs

```diff
@@ -0,0 +1,298 @@
+﻿//
+// DatePicker.cs: DatePicker control
+//
+// Author: Maciej Winnik
+//
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Globalization;
+using System.Linq;
+
+namespace Terminal.Gui.Views;
+
+/// <summary>
+/// The <see cref="DatePicker"/> <see cref="View"/> Date Picker.
+/// </summary>
+public class DatePicker : TextField {
+
+	private DataTable table;
```

Comment:
Use `_table` style for class private members. Thanks.

Reply:
Done